docker/scripts/dev_start.sh

```diff
@@ -321,6 +315,9 @@ function determine_gpu_use() {
 }
 
 function main() {
+    init
+    parse_arguments
```

Comment:
Have you tested this script before PR?
`parse_arguments` without "$@" should not work.